tanner/utils/sqlite_db_helper.py

```diff
@@ -60,7 +60,7 @@ def copy_db(self, src, dst, working_dir):
         cursor.executemany("INSERT INTO " + table_name + " VALUES(" +
                            inserted_string_patt + ")", inserted_data)
 
-    def create_query_map(self, working_dir, db_name, ):
+    def create_query_map(self, working_dir, db_name):
```

Comment:
This PR isn't related to this, please remove this change.

Reply:
I think we should remove this.